content/client-lib-development-guide/features.textile

```diff
@@ -58,7 +58,7 @@ h3(#restclient). RestClient
 * @(RSC16)@ @RestClient#time@ function sends a get request to @rest.ably.io/time@ and returns the server time in milliseconds since epoch or as a Date/Time object where suitable
 * @(RSC7)@ Sends REST requests over HTTP and HTTPS to the REST end-point @rest.ably.io@
 ** @(RSC7a)@ The header @X-Ably-Version: 0.8@ must be included in all REST requests to the Ably end-point
-* @(RSC18)@ If @ClientOptions#tls@ is true, then all communication is over HTTPS. If false, all communication is over HTTP however Basic Auth over HTTP will raise an exception as private keys cannot be submitted over an insecure connection. See @Auth@ below
+* @(RSC18)@ If @ClientOptions#tls@ is true, then all communication is over HTTPS. If false, all communication is over HTTP however "Basic Auth":https://en.wikipedia.org/wiki/Basic_access_authentication over HTTP will raise an exception as private keys cannot be submitted over an insecure connection. See @Auth@ below
```

Comment:
> raise an exception

This is environment-specific - I think we were preferring wording such as "result in an error" which is generic.